code/game/turfs/open/chasm.dm

```diff
@@ -112,3 +116,26 @@
 	underlay_appearance.icon = 'icons/turf/floors.dmi'
 	underlay_appearance.icon_state = "dirt"
 	return TRUE
+
+/turf/open/chasm/void
+	name = "endless void"
+	desc = "you feel as if you are watched."
+	baseturfs = /turf/open/chasm/void
+	var/destination
+
+/turf/open/chasm/void/chasm_destination()
+	if(destination)
+		return locate(loc.x,loc.y,destination)
+	var/list/zlist = SSmapping.z_list.Copy()
+	var/list/numlist = list()
+	zlist -= SSmapping.levels_by_trait(ZTRAIT_CENTCOM)
+	zlist -= SSmapping.levels_by_trait(ZTRAIT_POCKETDIM)
+	for(var/X in zlist)
+		var/datum/space_level/level = X
+		numlist += level.z_value
+	return locate(loc.x,loc.y,pick(numlist))
+
+/turf/open/chasm/void/lavaland/chasm_destination()
+	///I know this isnt always lavaland but shut the fuck up ok?
```

Comment:
it would clash with other things that could be on the reserved z level, so better to just keep it in it's own z level.